org.eclipse.draw2d.tests/src/org/eclipse/draw2d/test/swtbot/AbstractSWTBotTests.java

```diff
@@ -136,6 +147,20 @@ protected void doTest(Snippet annotation, Invocation<Void> statement) throws Thr
 		}
 	}
 
+	protected void scrollVertical(int x, int y, int count) {
```

Comment:
should have have a mousewheel in the name to make it clearer and distinguish from other scrolls (e.g., scrollbar)?